content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -15,7 +15,7 @@ This page shows you how to install [Minikube](/docs/tutorials/hello-minikube), a
 
 {{% capture prerequisites %}}
 
-VT-x or AMD-v virtualization must be enabled in your computer's BIOS.  To check this on Linux run the following and verify the output is non-empty:
+VT-x or AMD-v virtualization must be enabled in your computer's firmware settings.  To check this on Linux, run the following command and verify the output is non-empty:
 ```shell
 egrep --color 'vmx|svm' /proc/cpuinfo
```

Comment:
@sftim , in general, introducing the tabs per OS nicely cleans up the page. I also found that the formatting for the content in tabs does some unexpected rendering.
nit: This prerequisite section applies to all, but yet the user is instructed to verify only on Linux.

Reply:
I'll move the section about verifying virtualization to the Linux section, and I'll see if I give a similar advice for Windows.

I (or anyone else who's keen) might tackle the cleanup section in a separate PR.

Reply:
Updated the verification steps. @kbhawkey how does this look to you?